source/getting-started/sec-intro-to-sage.ptx

```diff
@@ -4,7 +4,7 @@
   <title>Intro to Sage</title>
 
   <p>
-    You can run and edit the Sage code directly in the SageMathCells embedded in this webpage. All the cells on the same page share the same memory. Be sure to run the cells in order. If you run the cells on a given page out of order, you may get unexpected results.
+    You can run and edit the Sage code directly in the SageMathCells embedded in this webpage. All the cells on the same page share the same memory. Be sure to run the cells in chronological order. If you run the cells on a given page out of order, you may get unexpected results.
```

Comment:
Not sure about chronological. "in the order that they are"? or something more natural...